e2e/yarn.lock

```diff
@@ -265,43 +265,43 @@
     chalk "^2.0.1"
```

Comment:
The large diff here feels like an error, since no packages have been added, modified or removed. Please revert this file or help me understand why it changed so dramatically.

Reply:
Reverted changes. Good eye, Greg. 🦅 I don't remember why and how. Thank you.